build/configs/artik053/tools/openocd/partition_gen.sh

```diff
@@ -36,8 +36,8 @@ OPENOCD_DIR_PATH=${BOARD_DIR_PATH}/tools/openocd
 FLASH_BASE=0x04000000
 
 # Partition information
-partsize_list=${CONFIG_ARTIK053_FLASH_PART_LIST}
-partname_list=${CONFIG_ARTIK053_FLASH_PART_NAME}
+partsize_list=${CONFIG_ARTIK053_FLASH_PART_LIST:="16,48,192,32,512,2400,1536,1536,1400,8,512,"}
```

Comment:
No. I can't do this. If default values are changed, we should modify here together. That is too unconvience. We should get them from Kconfig.